Comp2023-Imported/src/main/java/frc/robot/commands/DriveLimelightPath.java

```diff
@@ -58,7 +58,7 @@ public void execute( )
   {
     if (m_goalPose != null)
     {
-      m_swerve.driveWithPathFollowerExecute( );
+      m_swerve.driveWithPathFollowerExecute(true); //Sets Path to not check for current position

```

Comment:
While comments can be useful, I think that intellisense will display what that variable name is with "true" beside it. See what you think.